woocommerce/woocommerce
# Migrate Product Price to `block.json`

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




We want to migrate all of the blocks to `block.json` for consistency and to make sure all of them are correctly parsed by a script that creates Block References doc.

This PR migrates Product Price to `block.json`.

Part of https://github.com/woocommerce/woocommerce/issues/46917.

> [!NOTE]  
> `plugins/woocommerce-blocks/docs/block-references/block-references.md` is auto-generated hence contains more changes from other blocks.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Product Image smoke testing

1. Create new post
2. Add All Products block
3. Enter edit mode and make some edits in Product Price block inspector controls
4. Add Product Collection block
5. Choose any collection
6. Make some edits in Product Price block inspector controls
7. Save and go to frontend
8. Make sure the block is properly rendered, inspector controls settings are reflected



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
